projects/igniteui-angular/src/lib/query-builder/query-builder.component.ts

```diff
@@ -220,225 +160,68 @@ export class IgxQueryBuilderComponent implements AfterViewInit, OnDestroy {
     @Output()
     public expressionTreeChange = new EventEmitter();
```

Comment:
This gets typed as `EventEmitter<any>` which is really confusing to the consumer of the component.
Perhaps it can be typed as `EventEmitter<void>`

Reply:
Which also makes this documentation misleading:
![image](https://github.com/user-attachments/assets/135e755e-eac8-4071-9cb6-33cbcff37ed5)

It's not really two-way bindable if it doesn't emit a value.
It only has a change event that you can subscribe to.

What's more, attempting to two-way bind it, like in the docs, breaks it completely after the first update.